### PR TITLE
Fix UBD visualization code

### PR DESCRIPTION
**Pull request by lucasb-eyer**
This might or might not solve  #156. I believe it won't, but it's worth a try and fixes some latent bugs anyway.

### CODE REVIEW

**Comment by lucasb-eyer** [comment]
Do not merge yet, please

**Comment by lucasb-eyer** [comment]
Ok, ready for review/merge now.

File: upper_body_detector/src/main.cpp [hunk 1]
```diff
@@ -311,6 +327,9 @@ void connectCallback(message_filters::Subscriber<CameraInfo> &sub_cam,
 
 int main(int argc, char **argv)
 {
+    g_txt.push_back(QImage("/home/beyer/strands/0.png"));
+    g_txt.push_back(QImage("/home/beyer/strands/1.png"));
+    g_txt.push_back(QImage("/home/beyer/strands/2.png"));
```
Comment (cdondrup) [inline upper_body_detector/src/main.cpp]: I guess this will break for everyone else?

Comment (lucasb-eyer) [inline upper_body_detector/src/main.cpp]: Awww shit I was tired this is not supposed to be there, good catch!

Comment (cdondrup) [inline upper_body_detector/src/main.cpp]: I know the feeling ;)

**Comment by lucasb-eyer** [comment]
That's more what it should be like. We've been running with this for 2 days now and had no crash. But still, I don't really believe this is the crash's cause, so I wouldn't close the issue.

**Comment by cdondrup** [comment]
Tested on Linda, works fine for me.

**Pull request merged**